src/GitExtensions.PluginTemplate/GitExtensions.PluginTemplate.csproj.user

```diff
@@ -1,8 +1,12 @@
 ﻿<Project>
   <PropertyGroup>
-    <GitExtensionsDownloadPath>..\..\..\gitextensions.shared</GitExtensionsDownloadPath> <!-- path is relative to $(ProjectDir) -->
-    <GitExtensionsReferenceVersion>latest</GitExtensionsReferenceVersion> <!-- 'latest' or 'v3.1' (= tag from GitHub releases) or 'v3.1.0.5877' (= build number from AppVeyor)-->
-    <GitExtensionsReferenceSource>GitHub</GitExtensionsReferenceSource> <!-- 'GitHub' or 'AppYevor' -->
-    <GitExtensionsPath></GitExtensionsPath> <!-- for local builds (no download) -->
+    <!-- path is relative to $(ProjectDir) -->
+    <GitExtensionsDownloadPath>..\..\..\gitextensions.shared</GitExtensionsDownloadPath>
+    <!-- 'latest' or 'v3.1' (= tag from GitHub releases) or 'v3.1.0.5877' (= build number from AppVeyor)-->
+    <GitExtensionsReferenceVersion>latest</GitExtensionsReferenceVersion>
```

Comment:
It would be good to add a section to readme under Good Practices stating that `GitExtensionsReferenceVersion` used for local development should be synchronized with `GitExtensions.Extensibility` package version, so local dev loop and published package targets the same version of Git Extensions.

It's out of scope of this PR, so if you don't want to write it, I can do it in a separate PR

Reply:
I'm not so good on words, but I'll give it a try. 😀

Reply:
It also adds a dependency in the built nuget package and we use this dependency to validate that the package is intended for the current main app version (we still have gaps in this area, but in general 0.3 means targeting Git Extensions v4).

As we don't have a stable API yet, you typically need to "hack" into what Git Extensions give you in the current plugin API, and so it is important to build against "similar" version that you the plugin to run in (most of my plugins needs at least recompilation between main app version upgrades).

Using "latest" can be dangerous, because it may download even preview version.